src/elementRoleMap.js

```diff
@@ -10,39 +10,37 @@ type ElementARIARoleRelationMap = Map<ARIARoleRelationConcept, RoleSet>;
 
 const elementRoleMap: ElementARIARoleRelationMap = new Map([]);
 
-[...rolesMap.keys()]
-  .forEach((
-    key: ARIARoleDefintionKey,
-  ): void => {
-    const role = rolesMap.get(key);
-    if (role) {
-      [
-        ...role.baseConcepts,
-        ...role.relatedConcepts,
-      ]
-      .forEach((
-        relation: ARIARoleRelation,
-      ): void => {
-        if (relation.module === 'HTML') {
-          const concept = relation.concept;
-          if (concept) {
-            const conceptStr = JSON.stringify(concept);
-
-            let roles: ?RoleSet = ([...elementRoleMap.entries()]
-              .find(
-                // eslint-disable-next-line no-unused-vars
-                ([key, value]) => JSON.stringify(key) === conceptStr)|| []
-              )[1];
-
-            if (!roles) {
-              roles = new Set([]);
+for (const key: ARIARoleDefintionKey of [...rolesMap.keys()]) {
```

Comment:
The way I envisioned this is that we internally only deal with "records" (i.e. plain objects) and then export them as maps.

Then we should be able to iterate via 
```suggestion
Object.keys(roles).forEach((key) => {
```

Reply:
Unfortunately, with the current Babel configuration that would still require numerous transpilations for `Object.keys` and `forEach` and thus necessitate the dependency. As I understand it, all transpilations would have to be removed to remove the dependency. I understand that would be a consequence of supporting the older Node versions, but would mean that it is not possible to use any of the more modern API's such as `Object.keys`.

E.g. while it would be possible to remove many of the transpilation runtime artifacts, as long as `Map` and `Object.keys` remain somewhere in the bundle, the `babel` dependency remains.

Reply:
> Unfortunately, with the current Babel configuration that would still require numerous transpilations for Object.keys and forEach and thus necessitate the dependency.

Why do we transpile `Object.keys`?

Reply:
I think the Babel configuration is transpiling more features than might be desirable. I can look into this tomorrow when I am at work again. Do you by any chance have some ideas as to why Babel is doing more work than expected?

Reply:
I think the `browserslist` in the package.json might be too agressive. Try to find out which browsers it currently matches and then we might be able to relax it a bit.

Reply:
Yup, seems like `browserslist` does support setting a minimum Node version. Since this package supports Node 6+, #166 updates the `browserslist` to reflect that.

Reply:
@eps1lon Could you please take a look at #166 and let me know what you think? It appears that this package does not use any APIs that didn't exist in Node 6 yet, so we can remove the runtime altogether. It still transpiles the commonjs/ESM exports, but other than that it should be good to go 😄